src/base/static/components/place-detail/editor-bar.tsx

```diff
@@ -0,0 +1,105 @@
+/** @jsx jsx */
+import * as React from "react";
+import { css, jsx } from "@emotion/core";
+
+import mq from "../../../../media-queries";
+import { EditorButton } from "../atoms/buttons";
+import { translate } from "react-i18next";
+import i18next, { TranslationFunction } from "i18next";
+import constants from "../../constants";
+
+// Types were added to react-i18next is a newer version.
+// TODO: Use supplied types when we upgrade i18next deps.
+// See: https://github.com/i18next/react-i18next/pull/557/files
+type TransProps = {
```

Comment:
Hopefully it's ok to copy this from `map.tsx`, since we'll be using supplied types when https://github.com/mapseed/platform/pull/1341 is merged. Will try to get to that this week.